src/request/FeedResponse.ts

```diff
@@ -16,4 +16,7 @@ export class FeedResponse<TResource> {
   public get requestCharge(): number {
     return this.headers[Constants.HttpHeaders.RequestCharge];
   }
+  public get activityId(): number {
```

Comment:
Please something to our tests to make sure it is being set in single and cross partition scenarios.